local.yml

```diff
@@ -55,7 +55,7 @@ services:
       - ./interface/frontend/dist:/app/dist:cached
       - ./interface/frontend/static:/app/static:cached
       - ./interface/frontend/test:/app/test:cached
-      - ./.git/logs/HEAD:/HEAD:cached
+      - ./.git:/.git:cached
```

Comment:
Hm, the .git subdir is big - is there not a smaller subset we could use instead? Or use `logs/HEAD`?

Reply:
Actually we could use the `logs/HEAD` file and read the contents with some tweaking. Let me update that.